package.json

```diff
@@ -5,7 +5,7 @@
     "email": "azsdkteam@microsoft.com",
     "url": "https://github.com/Azure/ms-rest-js"
   },
-  "version": "0.9.0",
+  "version": "0.9.1",
```

Comment:
I think our publishing script will drop this patch version and use the commit-counting thing instead.

Reply:
Oh, right. Good point. I'll remove this.